### PR TITLE
Documentation for issue #279

### LINKED ISSUE #279: Document how to use docker gccrs to build full rust .exe
I tried docker gccrs.  It works.  To help others get up to speed faster, it would be useful to provide more examples to build full executables with gccrs:
```
cd ~/tryredox/
git clone https://github.com/Rust-GCC/gccrs.git
cd ~/tryredox/gccrs/gcc/testsuite/rust.test/compilable
docker pull philberty/gccrs

docker run --net host -v `pwd`:`pwd` -w `pwd` -i -t philberty/gccrs:latest gccrs --help

# To build an object file:
docker run --net host -v `pwd`:`pwd` -w `pwd` -i -t philberty/gccrs:latest gccrs -g -O2 -c type_infer1.rs -o type_infer1.o

# To build an executable:
docker run --net host -v `pwd`:`pwd` -w `pwd` -i -t philberty/gccrs:latest gccrs -g -O2 type_infer1.rs -o type_infer1
```

### PR DESCRIPTION
**Pull request by giorgosLiako**
Resolves #279 . Document how to use docker gccrs to build full rust .exe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,11 +88,23 @@ Or you can build your own image:
...
 $ docker run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp \
     gccrs-dev:latest gccrs -g -O2 -c \
     gcc/testsuite/rust.test/compilable/type_infer1.rs -o type_infer1.o
 ```
 
+If you want to build an executable file:
+```bash
+$ docker run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp \
+    gccrs-dev:latest gccrs -g -O2 \
+    gcc/testsuite/rust.test/compilable/type_infer1.rs -o type_infer1.o
```
Comment (philberty) [inline README.md:104]: I think in terms of compiling an example to an executable you should remove the .o on the output location so it does not confused as an object file

Comment (giorgosLiako) [reply README.md:104]: Yes , you are right ! I will make the change

**Comment by philberty** [comment]
Can you also add 2 more examples where you use -S to emit assembly so it would be -S -o output.s and another example the same as the executable example but uses -frust-dump-all to emit the debug outputs.

**Comment by giorgosLiako** [comment]
The 2 options were added too.

**Pull request merged**